tests/test/scheduler/test_function_client_server.cpp

```diff
@@ -32,14 +32,17 @@ TEST_CASE("Test sending MPI message", "[scheduler]")
 
     const char* user = "mpi";
     const char* func = "hellompi";
-    const faabric::Message& msg = faabric::util::messageFactory(user, func);
     int worldId = 123;
-    int worldSize = 2;
+    faabric::Message msg;
+    msg.set_user(user);
+    msg.set_function(func);
+    msg.set_mpiworldid(worldId);
+    msg.set_mpiworldsize(2);
+    faabric::util::messageFactory(user, func);
 
     scheduler::MpiWorldRegistry& registry = getMpiWorldRegistry();
-    scheduler::MpiWorld& localWorld = registry.createWorld(msg, worldId);
```

Comment:
Here we created the world _twice_, once through the registry, another one directly calling `create`.